hoomd/mpcd/geometry.py

```diff
@@ -56,6 +56,127 @@ def __init__(self, no_slip):
         self._param_dict.update(param_dict)
 
 
+class CosineChannel(Geometry):
+    r"""Serpentine (sinusoidal) channel.
+
+    Args:
+        amplitude (float): Amplitude of cosine.
+        wavenumber (float): Wavenumber of cosine.
```

Comment:
After having started to use this a little, I think it would be clearer if we rename this argument to either `length` or `repeat_length` (so that it is the length of one period/repeat of the cosine), and internally convert it to a wavenumber in the C++ geometry class. The same change should be made for the `CosineExpansionContraction`. Could you please make this update before review? Please let me know if you have any questions about how to do that.

Reply:
I've changed it to repeat_length, please let me know if you need anything else.